rlbot_gui/gui/overlay.html

```diff
@@ -0,0 +1,512 @@
+                top: 1% !important; 
+                right: 7% !important;
+            }
+            
+            #votes {
+                align: center;
+                padding-top: 1%;
+                display: flex;
+            }
+            
+            #votes > div {
+                flex: 1;
+                position: absolute;
+                height: 3px;
+                width: 311px;
+                border-style: solid;
+                border-width: 7px;
+            }
+            
+            #votes-blue {
+                border-color: blue;
+                background-color: blue;
+                left: 149px;
+            }
+            
+            #votes-orange {
+                border-color: orange;
+                background-color: orange;
+                right: 147px;
+            }
+            
+            .fade-in {
+                visibility: visible;
+                opacity: 1;
+                transition: opacity 1s linear;
+            }                
+             
+            .fade-out {
+                visibility: hidden;
+                opacity: 0;
+                transition: visibility 0s 1s, opacity 1s linear;
+            }                
+        </style>
+    </head>
+
+    <body>
+        <img src="/imgs/top.png" class="set hide-after-match" />
+        <img src="/imgs/socials.png" class="set" />
+        <img id="blue-action" class="set hide-after-match fade-out" height=64 width=64 />
+        <img id="orange-action" class="set hide-after-match fade-out" height=64 width=64 />
+    
+        <div id="names" class="hide-after-match">
+            <div class="name-div" id="name-blue">Blue Team</div>
+            <div id="name-divider"></div>
+            <div class="name-div" id="name-orange">Orange Team</div>
+        </div>
+        
+        <div id="votes" class="hide-after-match">
+            <div id="votes-blue"></div>
+            <div id="votes-orange"></div>
+        </div>
+
+        <img src="/imgs/votes.png" class="set hide-after-match" style="z-index: 5" />
+        
+        <div id="footer" class="hide-after-match">
+            <img id="logo" src="/imgs/logo.png" height=10% width=auto />
+        </div>
+        
+        <video id="goal-splash" class="set" autoplay muted>
+            <source src="/imgs/frame.webm" type="video/webm" />
+        </video>
+        
+        <img class="endgame-appear fade-out" src="/imgs/endgame.png" />
+        
+        <div id="endgame" class="endgame-appear fade-out">
+            
+        </div>
+        
+        <script type="text/javascript" src="https://gitcdn.xyz/repo/tmijs/cdn/master/latest/1.x/tmi.min.js"></script>
+        <script type="text/javascript">
+            // options: WinterTide, RLBot, Johnnyboi_i
+            // will be expanded later
+            SETTING = "RLBot"
+            EXPERIMENTAL_FEATURES = false
+            TWITCH_CHANNEL = "rlbotofficial"
+            FPS = 60
```

Comment:
This takes up a significant amount of CPU for me, and gui.py is only refreshing the data once per second. I changed this to 1 on mine and it performs much better.

Reply:
Odd, my CPU doesn't break a sweat. I'll keep this in mind, but the reason it's high is because with lower values the goal splash animation can appear up to a full second too late. As I suggested, we should probably set a config for this stuff.

Reply:
Ideally we want to do it like this:
```py
    eel.start('main.html', size=(1000, 800), block=False, callback=on_websocket_close, options=options,
              disable_cache=True)

    def refresh_gtp():
        global game_tick_packet
        game_tick_packet = packet_reader.get_packet()

    th = threading.Thread(target=refresh_gtp).start()
    while not should_quit:
        do_infinite_loop_content()
        eel.sleep(1.0)
    th.join()
```